src/next-pagesdir.tsx

```diff
@@ -177,10 +195,13 @@ export function createNextDsfrIntegrationApi(
                                 />
                             </>
                         )}
-                        {isProduction && (
+                        {isProduction && !isBrowser && (
```

Comment:
I guess this is for concistency with App router?

Reply:
Yes exactly!